src/dawn/IIR/IIR.proto

```diff
@@ -0,0 +1,314 @@
+/*===------------------------------------------------------------------------------*- proto -*-===*\
+ *                          _
+ *                         | |
+ *                       __| | __ ___      ___ ___
+ *                      / _` |/ _` \ \ /\ / / '_  |
+ *                     | (_| | (_| |\ V  V /| | | |
+ *                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+ *
+ *
+ *  This file is distributed under the MIT License (MIT).
+ *  See LICENSE.txt for details.
+ *
+\*===------------------------------------------------------------------------------------------===*/
+
+// @defgroup iir_proto IIR protobuf
+// @brief This is a IIR description for Google's protocol buffers library
+// <a href="https://developers.google.com/protocol-buffers/">Protobuf</a>.
+
+syntax = "proto3";
+
+package dawn.proto.iir;
+import "statements.proto";
+
+/* ===-----------------------------------------------------------------------------------------===*/
+//      Statement
+/* ===-----------------------------------------------------------------------------------------===*/
+
+// @brief Statement-Wrapper for the ASTStmt.
+//
+// This wrapper is used to have a common base for Stmts and Exprs
+message Statement {
```

Comment:
it looks like we dont need this level? It does not add anything to the ASTStmt

Reply:
it reflects the current structure of the code. Would you prefer it removed?

Reply:
it is gone